src/main.etk

```diff
@@ -29,10 +33,14 @@ jumpi
 # READ #
 ########
 
-# Check if calldata is less than 32 bytes
-push1 32        # [32]
-calldatasize    # [calldatasize, 32]
-lt              # [calldatasize < 32]
+# Check if calldata has the function signature
```

Comment:
Perhaps also check that calldata == 36 bytes long?

Reply:
Added!